incuna-sass/functions/_functions-variables.sass

```diff
@@ -0,0 +1,5 @@
+// Set a default font-base variable here that can
+// be overwritten locally
+// This way we can import incuna-sass files above
+// locally defined variables without getting errors
+$font-base: 16px !default
```

Comment:
I thought about doing that because (as I think we have discussed before) I'd like to define more variables in incuna-sass and it would be easier to have one variables file that contains them all but won't that get confusing? Especially if you can import sections of the library without also importing the variables. I don't know.